main.asm

```diff
@@ -9943,7 +9943,7 @@ Functione5bb: ; e5bb
 	ld bc, $0020
 	call AddNTimes
 	ld de, TempMonSpecies
-	ld bc, $0020
+	ld bc, PartyMon1Status - PartyMon1
```

Comment:
Equivalent to, but should be, `PartyMon1End - PartyMon1` (the end of the inner `box_struct`).